platforms/corda-ent/charts/notary-initial-registration/templates/job.yaml

```diff
@@ -167,27 +168,27 @@ spec:
             touch ${BASE_DIR}/certificates/done.txt
           volumeMounts:
           - name: notary-conf
-            mountPath: /opt/corda/CM-FILES/notary.conf
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/CM-FILES/notary.conf
             subPath: notary.conf
           - name: notary-conf
-            mountPath: /opt/corda/CM-FILES/notary-initial-registration.sh
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/CM-FILES/notary-initial-registration.sh
             subPath: notary-initial-registration.sh
           - name: notary-conf
-            mountPath: /opt/corda/CM-FILES/create-network-parameters-file.sh
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/CM-FILES/create-network-parameters-file.sh
             subPath: create-network-parameters-file.sh
           - name: notary-etc
-            mountPath: /opt/corda/etc
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/etc
           - name: notary-nodeinfo
-            mountPath: /opt/corda/additional-node-infos
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/additional-node-infos
           - name: notary-certificates
-            mountPath: /opt/corda/certificates
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/certificates
           - name: certificates
-            mountPath: /opt/corda/DATA
+            mountPath: {{ $.Values.nodeConf.volume.baseDir }}/DATA
           resources:
             requests:
-              memory: {{ .Values.cordaJarMx }}M
+              memory: {{ .Values.nodeConf.resources.requests }}

```

Comment:
`nodeConf.pod.resources.requests` was used in idman & signer, keep it consistent as well, same for line 191